test/relations.test.js

```diff
@@ -819,6 +819,18 @@ describe('relations', function() {
             done();
           });
         });
+        it('error case: filtering not done with wrong id key', function(done) {
```

Comment:
How about making the description consistent with the rest? something like:
`should return empty result when filtering with wrong id key`